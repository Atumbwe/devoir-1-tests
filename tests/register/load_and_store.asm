push 123
stack
load
push 3
stack
store
stack
