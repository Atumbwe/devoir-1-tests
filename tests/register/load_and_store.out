[ 123 ]
[ 3 ]
[ 3 123 ]
